magento/magento2
# Making configurable settings for MAX_IMAGE_WIDTH and MAX_IMAGE_HEIGHT

### Description

It was added a new system fieldset, where the max-width and max-height can be set.
`Stores / Configuration / Advanced / System / Images Configuration`

So these 2 values are more flexible now for those which need to have bigger images in their shops.


### Fixed Issues (if relevant)

1. magento/magento2#13747: Wysiwyg > Image Uploader >Max width/height
2. ...

### Manual testing scenarios
1. Get an image of 3000x700
https://dummyimage.com/3000x700
2. Open any wysiwyg editor
3. Upload an image
4. The uploaded image should have maximum configured size

### Contribution checklist
